akka-projection-slick/src/test/scala/akka/projection/slick/SlickOffsetStoreSpec.scala

```diff
@@ -57,12 +63,16 @@ class OffsetStoreSpec
   override implicit val patienceConfig: PatienceConfig =
     PatienceConfig(timeout = Span(3, Seconds), interval = Span(100, Millis))
 
-  val dbConfig: DatabaseConfig[H2Profile] = DatabaseConfig.forConfig("akka.projection.slick", OffsetStoreSpec.config)
+  private val slickConfig = SlickOffsetStoreSpec.config.getConfig(SlickSettings.configPath)
```

Comment:
I didn't want to make this test dependent on the ActorSystem just to read the config, so instead, I'm passing the config directly. 

To avoid copying "akka.projection.slick" everywhere, I added to a constant `SlickSettings.configPath`